testing/tests/DevExpress.ui.widgets/treeViewParts/virtualMode.js

```diff
@@ -1225,6 +1225,56 @@ QUnit.test('widget should support resolving promise if it is returned from the c
     assert.deepEqual(treeView.option('items'), [item], 'nodes were added after deferred is resolved');
 });
 
+QUnit.test('expandItem promise should be resolved if return value is empty array (T1114997)', function(assert) {
+    const done = assert.async();
+    const data = [{
+        id: 1,
+        parentId: null,
+        name: 'A',
+        expanded: false,
+    }, {
+        id: 2,
```

Comment:
do we really need second and third items? Do we need to load them in `createChildren`?
Seems like this test will be red even with the single element

Reply:
https://github.com/DevExpress/DevExtreme/pull/22630/commits/85101f8f4d750571687ef4ebb5532e64e1ceeb19